.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: "Testing"
+
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.8]
+        geomstats-backend : ['numpy',]
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build using Python ${{matrix.python-version}} and Backend ${{matrix.geomstats-backend}}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{matrix.python-version}}
+
+      - name: install
+        run:  |
+          # wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+          # bash miniconda.sh -b -p $HOME/miniconda
+          # export PATH="$HOME/miniconda/bin:$PATH"
+          # hash -r
+          # conda config --set always_yes yes --set changeps1 no
+          # conda update -q conda
+          # conda info -a
+          # conda create -q -n test-env python=3.8
+          # source activate test-env
+          pip install --upgrade pip setuptools wheel
+          pip install -q -r requirements.txt
+          
+          # # conda install -c conda-forge rdkit
+          # pip install biopython
+          # conda install -c conda-forge mdtraj
+          # conda install --yes -c plotly plotly-orca
+
+      - name: set Path 
+        run:  |
+          export PYTHONPATH=$PYTHONPATH:$(pwd)
+          jupyter --version
+
+      - name: run
+        env:
+          GEOMSTATS_BACKEND : ${{matrix.geomstats-backend}}
+        run: |
+          set -e
```

Comment:
@johmathe. wouldn't that be overkill. I removed the `for` loop and simplified to single line. Let me know if I should still move to script file

Reply:
@johmathe  on second note. turns out that jupyter nbconvert throws warning on empty files. instead of quietly exiting. 
`jupyter nbconvert --to notebook --execute !(submission-example-1)/*.ipynb`
 throws warning and run action fails. so I did move the script file